learn/core_concepts/documents.md

```diff
@@ -106,7 +106,7 @@ The above JSON document would look like this in NDJSON:
 
 #### CSV
 
-CSV files express data as a sequence of values separated by a delimiter character. Currently, Meilisearch **only supports the comma (`,`) delimiter**. Any [rules that apply to formatting CSV](https://datatracker.ietf.org/doc/html/rfc4180) also apply to Meilisearch documents.
+CSV files express data as a sequence of values separated by a delimiter character. By default, Meilisearch uses a single comma (`,`) as the delimiter. You can use the `csvDelimiter` query parameter of the [add, update, and replace documents endpoints](/reference/api/documents.md#add-or-update-documents) to set a different character. Any [rules that apply to formatting CSV](https://datatracker.ietf.org/doc/html/rfc4180) also apply to Meilisearch documents.
```

Comment:
```suggestion
CSV files express data as a sequence of values separated by a delimiter character. By default, Meilisearch uses a single comma (`,`) as the delimiter. You can use the `csvDelimiter` query parameter of the [update or replace documents endpoints](/reference/api/documents.md#add-or-update-documents) to set a different character. Any [rules that apply to formatting CSV](https://datatracker.ietf.org/doc/html/rfc4180) also apply to Meilisearch documents.
```

`add, update, and replace` make it seems like there is three way to add documents into meilisearch while in reality update and replace are both document addition